### PR TITLE
ET-3362 Created config template for creating aws budget with alert

### PR DESCRIPTION
**Pull request by carminetoriello**
## What 🕵️ 🔍

- What does the pull request add and/or solve?
This PR shows how to configure an AWS budget with alert by using Terraform

## References 📝 🔗

- [ET-3362](https://xainag.atlassian.net/browse/ET-3362)

## Reviewer checklist

*Reviewer agreement:*

* Reviewers shouldn't review a "Draft Pull Request", but rather wait for it to be marked as "Ready for Review".
* Reviewers do **not** commit or merge the pull request.
* If/when there are changes that need to be applied (applying Terraform or deploying Kubernetes manifests), the **assignee** is the one responsible for doing it.
* Use the labels `not-applied`, `partially-applied` and `fully-applied` so others know the current progress.
  * Remember to change labels accordingly once you start and finish to apply the changes.

### CODE REVIEW

File: aws/global/budget/budget.tf [hunk 1]
```diff
@@ -0,0 +1,15 @@
+resource "aws_budgets_budget" "cost" {
+  name         = "Initial budget"
```
Comment (Robert-Steiner) [inline aws/global/budget/budget.tf:2]:
```suggestion
  name         = "Development budget"
```

File: aws/global/budget/budget.tf [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  limit_amount = "50"
+  limit_unit   = "USD"
+  time_unit    = "MONTHLY"
+
+  notification {
+    comparison_operator        = "GREATER_THAN"
+    threshold                  = 80
+    threshold_type             = "PERCENTAGE"
+    notification_type          = "ACTUAL"
+    subscriber_email_addresses = ["carmine.toriello@xayn.com"]
```
Comment (Robert-Steiner) [inline aws/global/budget/budget.tf:13]: I will ask if we can have a devops email

**Review by Robert-Steiner — COMMENTED** [review]
Looks nice!
I just want to wait for a reply about the devops email

File: aws/global/budget/budget.tf (hunk 1 above)
Comment (carminetoriello) [reply aws/global/budget/budget.tf]: Yep, sounds better, done!
1f1d3deeac70dab68b83914d2f5f8ad7bf76e616

The change has been also applied on aws.

File: aws/global/budget/budget.tf (hunk 2 above)
Comment (carminetoriello) [reply aws/global/budget/budget.tf:13]: Awesome, thanks!

Comment (Robert-Steiner) [reply aws/global/budget/budget.tf:13]: you can use `devops@xayn.com`

Comment (carminetoriello) [reply aws/global/budget/budget.tf]: Done! 
2da4efe7f6d308c92174defdd78a9bd141b81242

Also applied on aws.

**Review by Robert-Steiner — APPROVED** [review]
👍

**Pull request merged**